packages/medusa/src/api-v2/store/collections/helpers.ts

```diff
@@ -0,0 +1,23 @@
+import { MedusaContainer } from "@medusajs/types"
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+
+export const refetchCollection = async (
```

Comment:
Similarly, once @riqwan merges the genric function I'll replace this one with it

Reply:
these are free to use now in develop